server/src/main/java/org/elasticsearch/script/Metadata.java

```diff
@@ -44,7 +44,7 @@ public class Metadata {
     protected static final String VERSION_TYPE = "_version_type";
     protected static final String VERSION = "_version";
     protected static final String TYPE = "_type"; // type is deprecated, so it's supported in the map but not available as a getter
-    protected static final String TIMESTAMP = "_now";
```

Comment:
I think this rename should be pulled out. It has nothing to do with protecting modification to ctx. If a dev is looking through git blame and sees this line was changed via this commit, it will be confusing what the two have to do with each other. Such a rename should be a simple and quick PR on its own.

Reply:
I think it's useful to have the context of removing the invalid `_timestamp` test, which is why this rename happens in this PR.

Reply:
As far as I can see, there is not rename of `_timestamp` here. There is a test removal, related to this change, and then there is this rename of variables. The latter has nothing to do with protecting modification to ctx.

Reply:
`TIMESTAMP` is not a good name because there is an old metadata value `_timestamp`, which was indicated in the test.

The only reason to change it was due to the weirdness of removing a `_timestamp` test while having a constant of value `_now` named `TIMESTAMP`.

Reply:
To be clear: I am not objecting to the rename. It seems like a fine change. But that rename could happen right now, in the main branch. It would be a trivial change, on its own, which could quickly be reviewed. But the commit message would then indicate what it is: a rename, for the purpose of clarification. The commit message in this PR does not indicate anything about the rename; it's unrelated to the change here. Someone doing git blame later and seeing this rename, perhaps trying to understand how this class has changed over time, would see this unrelated change and would likely be confused on how the two relate.

Reply:
Thinking about this more, we also expose `getTimestamp()` so this change made less sense without also changing `getTimestamp()`.

That interface change is worth a discussion, so I've reverted that rename and we can discuss it later.